app/Http/Controllers/CooperativeAdminController.php

```diff
@@ -92,12 +92,14 @@ public function update(Request $request, Cooperative $cooperative, $id)
         $validator = new UpdateUser();
         $data = $validator->execute($request, $admin);
 
-        DB::transaction(function () use (&$admin, $data, $request) {
+        try {
+            DB::beginTransaction();
+
             $admin->update($request->except('password', 'new_password', 'phones'));
 
             if (!empty($data['password'])) {
                 if (!Hash::check($data['password'], $admin->password)) {
-                    return response('Senha inválida.', 400);
+                    return response('', 400);
```

Comment:
Acho que eu tinha dito que esqueci de colocar uma messagem nesse response. Acho que pode manter como tava antes "Senha inávilda".